config/jobs/kubernetes/sig-release/release-branch-jobs/1.25.yaml

```diff
@@ -33,14 +33,14 @@ periodics:
           cpu: "1"
           memory: 3Gi
 - annotations:
-    fork-per-release-cron: 0 14-23/24 * * *
+    fork-per-release-cron: ""
     testgrid-alert-email: gke-kubernetes-accelerators-bugs@google.com
     testgrid-alert-stale-results-hours: "24"
     testgrid-dashboards: sig-release-1.25-blocking, google-gce
     testgrid-num-failures-to-alert: "6"
     testgrid-tab-name: gce-device-plugin-gpu-1.25
   cluster: k8s-infra-prow-build
-  cron: 0 8-23/24 * * *
+  cron: 0 14-23/24 * * *
```

Comment:
This run cadence change shouldn't happen now. It should only happen upon rotating jobs and that's going to happen when pushing 1.28 from k8s-beta to k8s-stable1. That said, I think we should revert changes to all affected files (`1.25.yaml` to `1.27.yaml`)

Reply:
Dropped the commit introducing those changes.